lib/rgeo/coord_sys/cs/entities.rb

```diff
@@ -339,15 +333,14 @@ def wkt_content(_)
       # Wolf parameters should be applied to geocentric coordinates, where
       # the X axis points towards the Greenwich Prime Meridian, the Y axis
       # points East, and the Z axis points North.
-
       class WGS84ConversionInfo < Base
-        def initialize(dx, dy, dz, ex, ey, ez, ppm) # :nodoc:
-          @dx = dx.to_f
-          @dy = dy.to_f
-          @dz = dz.to_f
-          @ex = ex.to_f
-          @ey = ey.to_f
-          @ez = ez.to_f
+        def initialize(dxm, dym, dzm, exas, eyas, ezas, ppm) # :nodoc:
```

Comment:
I don't understand the prefixes you chose here 😅

Reply:
I don't really know, but seems to me like `dx` is clearer than `dxm`.

Maybe the full length? `shift_x_meters`, `rotation_x_arc_seconds`

or compromise: `dx_meters`, `ex_arc_seconds`

Reply:
I could add `dx`, `dy`, `dz`, etc., to the allowed varables like `x`, `y`, etc., if that would work out better?

Reply:
@thestelz well I don't know, `ex`, `ex` and `ez` without context seem a bit hard to understand as well, don't you think?